text/0631-refresh-method-for-router-service.md

```diff
@@ -0,0 +1,51 @@
+- Start Date: 2020-05-23
+- Relevant Team(s): Ember.js
+- RFC PR: https://github.com/emberjs/rfcs/pull/631
+- Tracking: (leave this empty)
+
+# RouterService#refresh
+
+## Summary
+
+> Add a refresh method to the router service that calls refresh on all currently active routes.
+
+## Motivation
+
+> We want to be able to call refresh on all currently active routes from a centralized service
+or from a component. As a side benefit, we will be able to do this without relying on the send api,
+which we would like to deprecate. This enables us to get the latest data from the model hook.
```

Comment:
Maybe tweak this to not say "which we would like to deprecate" but say something like "which is being discussed as a possible deprecation in rfc#9999".